test/ValidateID.cpp

```diff
@@ -709,7 +709,7 @@ TEST_F(ValidateID, OpLoadResultTypeBad) {
 %5 = OpVariable %3 UniformConstant
 %6 = OpFunction %1 None %4
 %7 = OpLabel
-%8 = OpLoad %2 %5
+%8 = OpLoad %3 %5
```

Comment:
Whoops.  I didn't realize it was supposed to be a *BAD* result.
And the GitHub differ makes it very difficult to see beyond the end of this segment.